Sources/SwiftKuery/Table.swift

```diff
@@ -159,16 +288,50 @@ open class Table: Buildable {
         return self
     }
     
-    /// Add a primary key to the table.
-    ///
-    /// - Parameter columns: Columns that constitute the primary key.
+    /**
+     Function to set a single `Column` instances` as a primary key, in the `Table` instance.
```

Comment:
"Add a primary key, using the column which is passed in, to the table.

Calls the composite primary key function (this should be defined as a link) passing a single column."